scene.lua

```diff
@@ -35,7 +35,7 @@ end
 function Scene:gamepadreleased(joystick, button)
 end
 
-function love.gamepadaxis(joystick, axis, value)
+function Scene.gamepadaxis(joystick, axis, value)
```

Comment:
É isso mesmo. Só tem um typo aí que você também deixou passar:

```suggestion
function Scene:gamepadaxis(joystick, axis, value)
```